tests/src/test/java/org/eclipse/hono/tests/IntegrationTestSupport.java

```diff
@@ -1447,6 +1418,37 @@ public static void assertTelemetryMessageProperties(
             .mapEmpty();
     }
 
+    /**
+     * Creates a new AMQP sender link for publishing messages.
+     *
+     * @param endpoint The endpoint to send messages to.
+     * @param tenantId The tenant identifier.
+     * @return A future indicating the outcome.
+     * @throws NullPointerException if any of the parameters are {@code null}.
+     * @throws UnsupportedOperationException if the messaging network type is other than AMQP.
+     */
+    public Future<GenericSenderLink> createGenericMessageSender(final String endpoint, final String tenantId) {
```

Comment:
Maybe rename to `createGenericAmqpMessageSender`? Otherwise one might think this can also be used with Kafka.

Reply:
I have updated it.